src/components/state-details.js

```diff
@@ -271,17 +277,25 @@ export default function StateDetailsPage ({ location, data }) {
                   Mostly from different types of heating.
                 </p>
 
+                <p className="mt-5 mb-0">
+                  <img className="img-fluid" src={GasAppliances} title="Gas furnace, gas water heater, gas stove" alt="Gas furnace, gas water heater, gas stove" />
+                </p>
+
                 <p className="mt-5 mb-0">
                   {/* Sourced from Rewiring America Electrify Everything in Your Home guide */}
+
                   80% of the pollution of your typical home comes from heating your
                   space, water, and food.
                 </p>
               </div>
             </div>
 
             <p className="h3 mt-5">
-              To stop this pollution, we need to electrify our furnaces, water
-              boilers, and stoves.
+              To stop this pollution, we need to replace our furnaces with <a href="https://en.wikipedia.org/wiki/Heat_pump" target="_blank" rel="noreferrer">electric heat pumps</a>, electrify our water heaters, and cook with <a href="https://en.wikipedia.org/wiki/Induction_cooking" target="_blank" rel="noreferrer">induction and electricity</a> instead of gas.
```

Comment:
Can you break this onto multiple lines? At least personally I try to limit line lengths to ~80 or ~100